packages/graphql/src/classes/Node.ts

```diff
@@ -304,7 +304,10 @@ class Node extends GraphElement {
     }
 
     public getMainLabel(): string {
```

Comment:
I wouldn't remove this because we still treat the first label in the array different - specifically when creating indexes. I'm afraid this behaviour might be forgotten otherwise.

Reply:
Sounds good then!!